contracts/contracts/bridge/HomeBridge.sol

```diff
@@ -98,6 +98,41 @@ contract HomeBridge {
         }
     }
 
+    // check if a 2nd confirmTransfer would complete a transfer. this
+    // can happen after validator set changes.
+    function reconfirmCompletesTransfer(
```

Comment:
I guess it's more a question about Solidity patterns, but wouldn't it be better to have the confirm function return true or false, depending on if it was successful or not instead of having a separate function with separate logic to do the same check?

Reply:
I think a view function callable by **anyone** is useful. You have to call confirmTransfer as validator. Also it's non obvious what the return value should be.

Reply:
I believe you can fake the sender address for calls. But good point about the return value.